docs/contributing/documentation/myst-reference.md

```diff
@@ -350,6 +350,57 @@ print(f"my {a}nd line")
 ```
 ````
 
+### Tabs
+
+A simple tab component without synchronization, using a basic tab-set.
+
+::::{tab-set}
```

Comment:
Would you please change all the colons to backticks for consistency? I've done the first as an example.

Also let's use the `example` directive, which should provide both the source code and how it renders in a single block. It doesn't work with all Sphinx extensions, so if it doesn't work, we can remove it. Also remember to add a closing sequence of backticks for the outermost one I've added here.

``````suggestion
`````{example}
````{tab-set}
``````